assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/editor.scss

```
@@ -14,8 +14,13 @@
 
 .wc-block-components-checkbox {
```

> I wonder if, long term, the best solution is adding exceptions to the margin rule instead of maybe taking a more generic approach. For now, I think this is totally fine! 👍

I had the same thoughts, @sunyatasattva. What do you think about changing it to:

```suggestion
.wp-block-woocommerce-checkout-terms-block .wc-block-components-checkbox {
	margin-top: 0;
}
```

That way, the CSS definition only targets the element that you intend to change in #5191.